contracts/test/integration/utils/AllRolesConfig.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity =0.8.13;
+
+import {TribeRoles} from "contracts/core/TribeRoles.sol";
+
+contract AllRolesConfig {
```

Comment:
Really like this config of all roles, easy to see in one place what the role setup is